src/elements/toast/Toast.js

```diff
@@ -0,0 +1,56 @@
+import { Component, ElementRef } from 'angular2/core';
+import { CORE_DIRECTIVES } from 'angular2/common';
+
+@Component({
+    selector: 'novo-toast',
+    inputs: ['theme', 'icon', 'title', 'message', 'time', 'position'],
```

Comment:
I think we should let the user `*ngIf` for now but we can add a feature request issue for adding `show` as an input that is tied to `time`. I took off `time` and `position` on static `novo-toast` elements for now.